### PR TITLE
remove useless stuff

### PR DESCRIPTION
**Pull request by asbyth**
same thing as before but less useless changes

### CODE REVIEW

File: src/main/resources/assets/hyperium/sounds.json [hunk 1]
```diff
@@ -1,2079 +0,0 @@
-{
```
Comment (CoalCoding) [inline src/main/resources/assets/hyperium/sounds.json]: Are you sure this is unnecessary? I recall issues with sounds in the past. Have you built the client and installed it from the built jar to test it?

Comment (RDIL) [reply src/main/resources/assets/hyperium/sounds.json]: JSONLint isn’t happy about it.

Comment (asbyth) [reply src/main/resources/assets/hyperium/sounds.json]: tested anything i can, didnt break at all

File: src/main/java/cc/hyperium/config/Settings.java [hunk 2]
```diff
@@ -153,10 +153,6 @@
     @ToggleSetting(name = "gui.settings.uploadscreenshots")
     public static boolean DEFAULT_UPLOAD_SS = false;
 
-    @ConfigOpt(alt = "cc.hyperium.gui.settings.items.GeneralSetting;hideScoreboardNumbers")
```
[the remark this answers is not in the document]
Comment (asbyth) [reply src/main/java/cc/hyperium/config/Settings.java]: yes and it just doesnt do anything in the first place lol, i've never seen it function like it meant to

**Pull request merged**